src/Passes/Source/AllocationManager/IAllocationManager.hpp

```diff
@@ -0,0 +1,74 @@
+#pragma once
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "Llvm/Llvm.hpp"
+
+#include <memory>
+#include <string>
+#include <unordered_map>
+#include <vector>
```

Comment:
I have an impression that some of these headers are redundant for this file (in particular "Llvm/Llvm.hpp", <vector>, <unordered_map>).

Reply:
Yes, you are right. I've removed those that should not be there.